doc/internals/zarr-encoding-spec.rst

```diff
@@ -22,23 +22,58 @@ NetCDF and Zarr is that all NetCDF arrays have *dimension names* while Zarr
 arrays do not. Therefore, in order to store NetCDF data in Zarr, Xarray must
 somehow encode and decode the name of each array's dimensions.
 
```

Comment:
Could you also please update this section? In particular, the first paragraph suggests that futrue versions of the Zarr spec will include a more formal convention for the storage of the NetCDF data model in Zarr, but that future is already here with `dimension_names`! We only use an ad-hoc convention for storing data in Zarr v2.

Reply:
updated